startup_assistance/indico_startup_assistance/__init__.py

```diff
@@ -0,0 +1,13 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from indico.util.i18n import make_bound_gettext
+
+
+_ = make_bound_gettext('startup_assistance')
```

Comment:
Wouldn't something like `room_assistance` be a better name for the plugin?

Reply:
Hm, could be. I just sticked to how it was called before.